newton/_src/geometry/utils.py

```
@@ -131,6 +189,99 @@ def compute_obb(vertices: nparray) -> tuple[wp.transform, wp.vec3]:
+    if len(vertices) == 0:
+        return wp.transform_identity(), wp.vec3(0.0, 0.0, 0.0)
+
+    if len(vertices) == 1:
+        return wp.transform(wp.vec3(vertices[0]), wp.quat_identity()), wp.vec3(0.0, 0.0, 0.0)
+
+    # Step 1: Compute convex hull
+    hull_vertices, hull_faces = remesh_convex_hull(vertices, maxhullvert=0)  # 0 = no limit
+    hull_indices = hull_faces.flatten()
+
+    # Step 2: Compute mesh inertia
+    mass, com, inertia_tensor, volume = compute_mesh_inertia(
```

Ruff (0.13.1)

Unpacked variable `mass` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)